From: Luciano Bello <luciano@debian.org>
Date: Thu, 12 May 2016 23:24:44 +0200
Subject: Disable the coders EPHEMERAL, URL, HTTPS, MVG, MSL, TEXT, SHOW, WIN, and PLT
 Following the recomendation from imagetragick.com this patch 
 disable the coders EPHEMERAL, URL, HTTPS, MVG, MSL, TEXT, SHOW, WIN,
 and PLT since they are vulnerable to multiple attacks. These
 vulnerability were collectible called "ImageTragick".
Bug: https://www.imagemagick.org/discourse-server/viewtopic.php?f=4&t=29588
Bug-CVE: CVE-2016-3714, CVE-2016-3715, CVE-2016-3716, CVE-2016-3717, CVE-2016-3718
Bug-Debian: https://bugs.debian.org/823542
Last-Update: 2016-05-12
---
Index: imagemagick-6.6.9.7/config/policy.xml
===================================================================
--- imagemagick-6.6.9.7.orig/config/policy.xml	2016-06-01 13:18:17.623193827 -0400
+++ imagemagick-6.6.9.7/config/policy.xml	2016-06-01 13:18:17.619193774 -0400
@@ -54,4 +54,13 @@
   <!-- <policy domain="resource" name="thread" value="4"/> -->
   <!-- <policy domain="resource" name="throttle" value="0"/> -->
   <!-- <policy domain="resource" name="time" value="3600"/> -->
+  <policy domain="coder" rights="none" pattern="EPHEMERAL" />
+  <policy domain="coder" rights="none" pattern="URL" />
+  <policy domain="coder" rights="none" pattern="HTTPS" />
+  <policy domain="coder" rights="none" pattern="MVG" />
+  <policy domain="coder" rights="none" pattern="MSL" />
+  <policy domain="coder" rights="none" pattern="TEXT" />
+  <policy domain="coder" rights="none" pattern="SHOW" />
+  <policy domain="coder" rights="none" pattern="WIN" />
+  <policy domain="coder" rights="none" pattern="PLT" />
 </policymap>
